client/gutenberg/editor/style.scss

```diff
@@ -74,6 +74,10 @@ $gutenberg-theme-toggle: #11a0d2;
 		}
 	}
 
+	.editor-block-mover__control {
+		padding: 0;
```

Comment:
This CSS rule is already in `@wordpress/editor`, but `@wordpress/components` is overriding it. So maybe we don't need this if we import the `@wordpress/components` styles before the `@wordpress/editor` styles [here](https://github.com/Automattic/wp-calypso/blob/a34d137c6195b2b4037612f460bfa6343f68769c/assets/stylesheets/sections/gutenberg-editor.scss#L19).

Reply:
Nice one @mmtr!
Reordering the imports don't seem to change anything, and also made me notice a whole other thing. I'll update this ASAP!